onnxscript/test/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -365,13 +382,16 @@ def test_output_match(self, device: str, dtype: torch.dtype, op):
                     rtol = None
                     atol = None
 
-                # Use torch testing to ensure dtypes and shapes match
-                torch.testing.assert_close(
-                    torch.tensor(function_output),
-                    output_torch,
-                    rtol=rtol,
-                    atol=atol,
-                )
+                if isinstance(output_torch, bool):
+                    assert(output_torch == function_output)
```

Comment:
@justinchuby need you to check if this change is reasonable. it is for aten_equal, output_torch is just bool True/False, but  function_output will be tensor(array[True]) or tensor(array[False]), so the torch.testing.assert_close() will always fail.

Reply:
I think that makes sense. Which test case caused this?

Reply:
How about: calling

torch.tensor(output_torch) 

for all torch outputs to force them into a tensor? This way we don’t have conditionals in the code.